packages/suite/src/actions/suite/storageActions.ts

```diff
@@ -67,7 +67,7 @@ export const saveCoinjoinAccount =
 const removeCoinjoinRelatedSetting = (state: AppState) => {
     const settings = { ...state.suite.settings };
 
-    settings.isCoinjoinCexWarningHidden = false;
+    settings.isCoinjoinReceiveWarningHidden = false;
```

Comment:
This deserves migration, I guess.

Reply:
I was thinking about it but felt too granular to me. A very small amount of users who have their CJ account remembered would have to dismiss it again but that's it. If you think it's needed I will add a migration :)

Reply:
but the value `isCoinjoinCexWarningHidden` will remain in their storage until reset, but ok, maybe it's fine in this case and not needed to add the migration